dist/leaflet.distortableimage.js

```diff
@@ -1391,6 +1391,7 @@ L.KeymapperIconSet = L.IconSet.extend({
 /* eslint-disable max-len */
 L.ToolbarIconSet = L.IconSet.extend({
   _symbols:
+    '<symbol viewBox="0 0 18 18" id="cancel"><path d="M13.279 5.779l-1.058-1.058L9 7.942 5.779 4.721 4.721 5.779 7.942 9l-3.221 3.221 1.058 1.058L9 10.057l3.221 3.222 1.058-1.058L10.057 9z"/></symbol>' +
```

Comment:
Looks like we keep these in alphabetical order so please move `cancel` after `border_outer`

Reply:
:+1: